ko/documentation/installation/index.md

```diff
@@ -136,7 +136,7 @@ $ brew install ruby
 {: #solaris}
 
 Solaris 8에서 Solaris 10까지 [Sunfreeware][sunfreeware]에서 루비 1.8.7을 사용 가능하며
-[Blastwave][blastwave]에서 루비 1.8.7을 사용 가능합니다.
+Blastwave에서 루비 1.8.7을 사용 가능합니다.
```

Comment:
@ruby/www-ruby-lang-org-i18n-ko could you help here 🙏

Reply:
This would be okay:
``` diff
-Solaris 8에서 Solaris 10까지 [Sunfreeware][sunfreeware]에서 루비 1.8.7을 사용 가능하며
-Blastwave에서 루비 1.8.7을 사용 가능합니다.
+Solaris 8에서 Solaris 10까지 [Sunfreeware][sunfreeware]에서 루비 1.8.7을 사용 가능합니다.
```